blob/service.go

```diff
@@ -198,8 +212,12 @@ func (s *Service) getByCommitment(
 
 	namespacedShares, err := s.shareGetter.GetSharesByNamespace(ctx, header.DAH, nID)
 	if err != nil {
-		return nil, nil, err
+		if strings.Contains(err.Error(), share.ErrNotFound.Error()) {
```

Comment:
```suggestion
		if errors.Is(err, share.ErrNotFound) {
```

Reply:
Commenting this as a reminder to update to include share.ErrNamespaceNotFound once merged into main

In sync we said both can be converted to a BlobNotFound, but after a little thinking the error handling should probably be to return empty data for NamespaceNotFound with a nil error